homeassistant/components/modbus/switch.py

```diff
@@ -200,7 +200,6 @@ def __init__(self, hub: ModbusHub, config: Dict[str, Any]):
 
     def turn_on(self, **kwargs):
         """Set switch on."""
-
```

Comment:
This has nothing to do with changing constants (which I see as a good idea), the empty line is design,  please remove this change.